tensorflow/python/ops/parallel_for/BUILD

```diff
@@ -89,6 +89,7 @@ py_library(
         "//tensorflow/python:util",
         "//tensorflow/python/eager:context",
         "//tensorflow/python/eager:function",
+	"//tensorflow/python/ops/signal",
```

Comment:
Needs two spaces of indent (and in the other BUILD file too).

Build file auto-formatting works pretty well (much better than for Python): https://github.com/bazelbuild/buildtools/blob/master/buildifier/README.md

I believe we have a test that checks the BUILD file matches Buildifier's formatting, so usually best to run it.

Reply:
Fixed it.